gpflow/_version.py

```diff
@@ -13,4 +13,4 @@
 # limitations under the License.
 
 
-__version__ = "1.0.0"
+__version__ = "1.1.0dev"
```

Comment:
Do we want this version number?

Reply:
Don't know. I was thinking that it may be a good idea to have a version number for the working version of master that indicates that it hasn't been released yet. We need to increase the version number for this feature, as it changes the interface to certain models (`model.Z` is changed).

However, I'm sure that more features will be added before 1.1 is actually released?